<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerial">
    <property name="title">
     <string>Serial</string>
    </property>
    <addaction name="serialInfoAction"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="preferencesAction"/>
   </widget>
   <addaction name="menuPreferences"/>
   <addaction name="menuSerial"/>
  </widget>
  <widget class="QDockWidget" name="logDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="appLogLbl">
         <property name="text">
          <string>Application Log:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="appLogTextEdit">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="serialLogLbl">
         <property name="text">
          <string>Serial Log:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="serialLogTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="serialInfoAction">
   <property name="text">
    <string>Info...</string>
   </property>
  </action>
  <action name="preferencesAction">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
